recipes/libtool/all/conanfile.py

```diff
@@ -220,22 +202,36 @@ def package_info(self):
             if self.settings.os == "Linux":
                 self.cpp_info.system_libs = ["dl"]
 
+        # Use ACLOCAL_PATH to access the .m4 files provided with libtool
+        aclocal_path = os.path.join(self.package_folder, "res", "aclocal")
+        self.buildenv_info.append_path("ACLOCAL_PATH", aclocal_path)
+
+        # Everything below can be eliminated when Conan 1.x support is dropped
         bin_path = os.path.join(self.package_folder, "bin")
-        self.output.info("Appending PATH env: {}".format(bin_path))
+        self.output.info(f'Appending PATH env: {bin_path}')
         self.env_info.PATH.append(bin_path)
 
         bin_ext = ".exe" if self.settings.os == "Windows" else ""
 
-        libtoolize = tools.unix_path(os.path.join(self.package_folder, "bin", "libtoolize" + bin_ext))
-        self.output.info("Setting LIBTOOLIZE env to {}".format(libtoolize))
+        libtoolize = os.path.join(self.package_folder, "bin", "libtoolize" + bin_ext)
+        libtoolize = "/" + libtoolize.replace("\\", "/").replace(":", "") # Can't use unix_path with Conan 2.0
+        self.output.info(f'Setting LIBTOOLIZE env to {libtoolize}')
         self.env_info.LIBTOOLIZE = libtoolize
 
-        for key, value in self._libtool_relocatable_env.items():
-            self.output.info("Setting {} environment variable to {}".format(key, value))
+        libtool_relocatable_env = {
+            "LIBTOOL_PREFIX": self.package_folder,
+            "LIBTOOL_DATADIR": os.path.join(self.package_folder, "share"),
+            "LIBTOOL_PKGAUXDIR": os.path.join(self.package_folder, "share", "libtool", "build-aux"),
+            "LIBTOOL_PKGLTDLDIR": os.path.join(self.package_folder, "share", "libtool"),
+            "LIBTOOL_ACLOCALDIR": os.path.join(self.package_folder, "share", "aclocal"),
+        }
+
+        for key, value in libtool_relocatable_env.items():
+            self.output.info(f'Setting {key} environment variable to {value}')
             setattr(self.env_info, key, value)
 
-        libtool_aclocal = tools.unix_path(os.path.join(self._datarootdir, "aclocal"))
-        self.output.info("Appending ACLOCAL_PATH env: {}".format(libtool_aclocal))
-        self.env_info.ACLOCAL_PATH.append(libtool_aclocal)
-        self.output.info("Appending AUTOMAKE_CONAN_INCLUDES environment variable: {}".format(libtool_aclocal))
-        self.env_info.AUTOMAKE_CONAN_INCLUDES.append(libtool_aclocal)
+        aclocal_path = "/" + aclocal_path.replace("\\", "/").replace(":", "") # Can't use unix_path with Conan 2.0
```

Comment:
we have added a helper function to cover cases were `unix_path` was erroneously used in `package_info`, to aid with the migration -

Reply:
Hi @jcar87,
Okay--I replaced the above line with a call to unix_path_package_info_legacy. Thanks